job-dsls/jobs/srcclr_scan_pipeline.groovy

```diff
@@ -0,0 +1,74 @@
+import org.kie.jenkins.jobdsl.Constants
+
+def prefix = Constants.GITHUB_REPO_PREFIX
+def repoListFileId = Constants.REPO_LIST_FILE_ID
+
+def pipelineScript =
+        '''
+@Library('jenkins-pipeline-shared-libraries')_
+
+def repoList = []
+node('kie-rhel7') {
+    stage('Read repo file') {
+        configFileProvider([configFile(fileId: "${REPO_LIST_FILE_ID}", variable:'repoListFile')]) {
+            def repoFile = readFile "$repoListFile"
+            repoList = repoFile.readLines()
+        }
+    }
+}
+
+def branches = [:]
+
+for (repo in repoList) {
+    def branchName = 'Source Clear ' + "${repo}"
+    println 'Repo before node ' + "${repo}"
+    def repoName = "${repo}"
+    branches[branchName] = {
+        node('kie-rhel7') {
+            stage(branchName) {
+                def url = "${URL_PREFIX}" + "${repoName}"
```

Comment:
it can be replaced by `def url = "${URL_PREFIX}${repoName}"`

Reply:
even if that works the same way, I am not against it ;)